main_pullrequest.go

```diff
@@ -94,25 +110,27 @@ func processGitHubPullRequest(
 					pr.Sender.GetLogin(),
 				)
 			}
-			err = retryOnError(retryParams{
-				retryFunc: func() error {
-					return startPRPipeline(log, prBranchName, pr, conf, isOrgMember)
-				},
-				compFunc: func(compareError error) bool {
-					re := regexp.MustCompile("Missing CI config file|" +
-						"No stages / jobs for this pipeline")
-					switch {
-					case compareError == nil:
-						return noRetry
-					case re.MatchString(compareError.Error()):
-						log.Infof("start client pipeline for PR '%d' is skipped", pr.Number)
-						return noRetry
-					default:
-						log.Errorf("failed to start client pipeline for PR: %s", compareError)
-						return doRetry
-					}
-				},
-			})
+			if !skipCI {
```

Comment:
perhaps add above ```if prRef != "" && !skipCI {...```?

Reply:
`prRef != ""` is already covered by the parent if branch.

Reply:
I meant to add the option check to the if `if prRef != "" {` above :)